src/calc-length.js

```diff
@@ -64,6 +64,21 @@
 
   CalcLength.prototype = Object.create(shared.LengthValue.prototype);
 
+  // Length Calculation Methods
+  CalcLength.prototype.multiply = function(multiplier) {
+    var calcDictionary = new Object;
+
+    //iterate through all length types and multiply all non null lengths 
+    for(var index in shared.LengthValue.LengthType){
```

Comment:
It's probably less confusing (and more common) to write

for (var i = 0; i < shared.LengthValue.LengthType.length; i++) {
var type = shared.LengthValue.LengthType[i];
...etc...
}